### PR TITLE
update Kraft Docker Compose file to use updated environment variables

### PR DESCRIPTION
**Pull request by davetroiano**
As part of this change, we no longer need the update_run.sh script to get Kraft mode working in Docker.

### CODE REVIEW

File: cp-all-in-one-kraft/docker-compose.yml [hunk 1]
```diff
@@ -20,15 +21,11 @@ services:
...
       KAFKA_PROCESS_ROLES: 'broker,controller'
-      KAFKA_NODE_ID: 1
       KAFKA_CONTROLLER_QUORUM_VOTERS: '1@broker:29093'
       KAFKA_LISTENERS: 'PLAINTEXT://broker:29092,CONTROLLER://broker:29093,PLAINTEXT_HOST://0.0.0.0:9092'
       KAFKA_INTER_BROKER_LISTENER_NAME: 'PLAINTEXT'
       KAFKA_CONTROLLER_LISTENER_NAMES: 'CONTROLLER'
       KAFKA_LOG_DIRS: '/tmp/kraft-combined-logs'
-    volumes:
-      - ./update_run.sh:/tmp/update_run.sh
-    command: "bash -c 'if [ ! -f /tmp/update_run.sh ]; then echo \"ERROR: Did you forget the update_run.sh file that came with this docker-compose.yml file?\" && exit 1 ; else /tmp/update_run.sh && /etc/confluent/docker/run ; fi'"
```
Comment (stejani-cflt) [inline cp-all-in-one-kraft/docker-compose.yml:31]: Do you think we can use this same bash script mechanism to generate the cluster id?

**Review by stejani-cflt — APPROVED** [review]

File: cp-all-in-one-kraft/docker-compose.yml (hunk 1 above)
Comment (davetroiano) [reply cp-all-in-one-kraft/docker-compose.yml:31]: done

**Review by bbejeck — APPROVED** [review]
LGTM

**Review by stejani-cflt — APPROVED** [review]
LGTM

**Pull request merged**